doc/user/QUICKSTART.md

```diff
@@ -0,0 +1,88 @@
+# Quickstart Guide
+
+This guide will help you get started using F3D.
+
+As an overview, you'll learn how to run F3D and open files, configure basic scene constructions, interact with them, and play animations.
+
+## Prerequisites
+
+To use F3D, you'll need to install F3D. See the [Installation](INSTALLATION.md) page for the latest release package supported by your system.
+
+## Running F3D
+
+Once F3D has been installed, you should be able to open a file in any of the following ways:
+
+* Open a file directly from your file manager.
+* Open F3D, then drag and drop a file into the application window.
+* Run F3D from the terminal with a set of command-line [options](OPTIONS.md).
+* Configure F3D as a [thumbnailer](DESKTOP_INTEGRATION.md) for supported file formats with certain file managers.
+
+If you choose to run F3D from the terminal, you have different options.
+
+1\. Open a file and visualize it interactively:
+
+```
+f3d /path/to/file.ext
+```
+
+2\. Open a file and save the rendering into an image file:
+
+```
+f3d /path/to/file.ext --output=/path/to/img.png
+```
+
+If you need help, specify the `--help` [option](OPTIONS.md):
+
+```
+f3d --help
+man f3d # Linux only
+```
```

Comment:
these 3 examples are exactly the same as the "very quick start" in the readme.md, 
 - should it be expanced  a bit here ?
 - should readme.md examples be a bit reduced ?
 
 I'm not  sure, just asking

Reply:
slightly shortened the readme.md section for now.